src/Language/PureScript/Publish.hs

```diff
@@ -204,6 +208,9 @@ getBowerInfo = either (userError . BadRepositoryField) return . tryExtract
           (Left (BadRepositoryType repositoryType))
         maybe (Left NotOnGithub) Right (extractGithub repositoryUrl)
 
+bowerLicenseExists :: PackageMeta -> Bool
+bowerLicenseExists pkgMeta = bowerLicence pkgMeta == []
```

Comment:
It seems to me like this does the opposite from what the function name hints at; with this name, I'd expect the implementation to be `not . null . bowerLicence`